site/content/docs/concepts/environments.en.md

```diff
@@ -29,9 +29,9 @@ When you first create a new environment, no services are deployed to it. To depl
 
 ### VPC and Networking
 
-Each environment gets its own multi-AZ VPC. Your VPC is the network boundary of your environment, allowing the traffic you expect in and out, and blocking the rest. The VPCs Copilot creates are spread across two availability zones to help balance availability and cost - with each AZ getting a public and private subnet.
+Each environment gets its own multi-AZ VPC. Your VPC is the network boundary of your environment, allowing the traffic you expect in and out, and blocking the rest. The VPCs Copilot creates are spread across two availability zones, with each AZ getting a public and private subnet, following [AWS best practices](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-security-best-practices.html). Your services are launched in the public subnets, to help manage costs. Services can be reached only through your load balancer.
 
-Your services are launched in the public subnets but can be reached only through your load balancer.
+Workload subnet placement can be modified using the `network.vpc.placement` field in the AWS Copilot CLI manifest. For additional details, see this field in the [Backend Service](../manifest/backend-service.en.md), [Load Balanced Web Service](../manifest/lb-web-service.en.md), [Request-Driven Web Service](../manifest/rd-web-service.en.md), [Scheduled Job](../manifest/scheduled-job.en.md), or [Worker Service](../manifest/worker-service.en.md) manifest specifications.
```

Comment:
nit: It's probably okay to just link to one of them so that we don't have to remember updating this section.
```suggestion
Workload subnet placement can be modified using the [`network.vpc.placement`](../manifest/lb-web-service.en.md#network-vpc-placement) field in the manifest.
```